wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,73 @@
-Why am I banned?
-----------------
+# Handling Cheating
```

Comment:
Rename the whole article to "Handling Foul Play". Requires changing folder name too but I don't know how to do that ?

Reply:
Simply rename the folder, lol.